spyder/plugins/editor/fallback/fallback.py

```diff
@@ -0,0 +1,40 @@
+
+# -*- coding: utf-8 -*-
+
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Fallback completion actor.
+This plugin takes a plain text/source file and returns the individual words
+written on it.
+"""
+
+import re
+import logging
+from queue import Queue
+from threading import Thread, Lock
+
+logger = logging.getLogger(__name__)
+
+# Get all valid tokens that start by a letter and are
+# followed by a sequence of letters, numbers or underscores of length > 0
+regex = re.compile(r'[a-zA-Z]\w+')
```

Comment:
Hmmm this might be extended, since for instance on CSS names usually use `-`

Reply:
We could add ``-`` to the pattern, but I'm wondering what would happen in this scenario: ``var1 = var2-var3``, a subtraction without whitespace around the operator

Reply:
Another solution would be to define another regex for languages/plain text that allow ``kebab-case`` (CSS, LISP, Perl) and other for the rest of languages that use ``snake_case`` or ``camelCase``

Reply:
Hmmm well for now I would just check if it is CSS os SCSS or HTML or XML ?